src/HealthChecks.OpenIdConnectServer/HealthChecks.OpenIdConnectServer.csproj

```diff
@@ -5,7 +5,7 @@
     <PackageTags>$(PackageTags);IdentityServer;IdSvr;OpenId;OpenIdConnect</PackageTags>
     <Description>HealthChecks.OpenIdConnectServer is the health check package for OpenIdConnect servers</Description>
     <VersionPrefix>$(HealthCheckOpenIdConnectServer)</VersionPrefix>
-    <RootNamespace>HealthChecks.IdSvr</RootNamespace> <!--For backward naming compatibility-->
+    <RootNamespace>HealthChecks.OpenIdConnectServer</RootNamespace> <!--For backward naming compatibility-->
```

Comment:
I think that this line should be simply deleted now. (RootNamespace is by default the same as project name)

```suggestion
```